harness/determined/pytorch/_pytorch_context.py

```diff
@@ -459,33 +473,31 @@ def configure_apex_amp(
         if not enabled or not self.env.managed_training:
             return models, optimizers
 
-        if self._scaler is not None:
-            check.false(self._scaler.is_enabled(), "Do not mix APEX with PyTorch AMP")
-
-        check.false(self._use_apex, "Please only call configure_apex_amp once.")
-        if self.distributed.size > 1:
-            check.eq(
-                num_losses,
-                1,
-                "When using parallel/distributed training, "
-                "Determined only supports configure_apex_amp with num_losses = 1",
-            )
+        if self._scaler is not None and self._scaler.is_enabled():
+            raise det.errors.InvalidExperimentException("Do not mix APEX with PyTorch AMP.")
 
-        self._use_apex = True
+        if self._use_apex:
+            raise det.errors.InvalidExperimentException("Please only call configure_apex_amp once.")
 
         if self.distributed.size > 1:
-            check.eq(
-                self._aggregation_frequency,
-                1,
-                "Mixed precision training (AMP) is not supported with "
-                "distributed training and "
-                "aggregation frequency > 1.",
+            if num_losses != 1:
+                raise det.errors.InvalidExperimentException(
+                    "When using parallel/distributed training, "
```

Comment:
Please change this to `When using distributed training` ("parallel" training was a term we used to use to describe multi-slot, single-node training, but we dropped it a longtime ago)

Reply:
Done.